src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicSmokeTestTransformer.java

```diff
@@ -145,8 +149,13 @@ private SmokeTestClassView generateSmokeTestCsproj(GapicInterfaceContext context
   }
 
   private StaticLangApiMethodView createSmokeTestCaseApiMethodView(MethodContext methodContext) {
-    StaticLangApiMethodView initialApiMethodView =
-        new CSharpApiMethodTransformer().generateFlattenedMethod(methodContext);
+    MethodConfig methodConfig = methodContext.getMethodConfig();
+    StaticLangApiMethodView initialApiMethodView;
+    if (methodConfig.isPageStreaming()) {
```

Comment:
Conditional operator?

Reply:
Further `if`s will be required for support of things like operation-based results.
So I'd rather leave it as it is.